spec.md

```diff
@@ -1003,47 +1011,39 @@ The following example shows an inferred reset that will get inferred to a synchr
 ``` firrtl
 input a : UInt<1>
 wire reset : Reset
-connect reset, a
+connect reset, asReset(a, Reset<sync, high>)
```

Comment:
This change is fine. But if we move forward with this change, remind me to make a pass against all of our examples to clean them up using a native `Reset<_, _>` type rather than casting `UInt<1>`s.

Reply:
While this would need to happen, there is a transition period where we will likely want to codegen casts different (or accept them for a while).